backend/model/db.js

```diff
@@ -59,6 +62,15 @@ Follow.belongsTo(User, {as: 'followed', foreignKey: 'follower_user_id', onDelete
 User.hasMany(Follow, {as: 'following', foreignKey: 'followed_user_id', onDelete: 'CASCADE'})
 Follow.belongsTo(User, {as: 'following', foreignKey: 'followed_user_id', onDelete: 'CASCADE'})
 
+User.hasMany(Event, {as: 'eventUser', foreignKey: 'userId', onDelete: 'CASCADE'})
+Event.belongsTo(User, {as: 'eventUser', foreignKey: 'userId', onDelete: 'CASCADE'})
```

Comment:
what is the use of "as" parameter here? does it not work without alias?

Reply:
it probably works, it does give some context, I followed the previous usage for consistency, dont think that it is important.

Reply:
it makes writing the code a tad harder.